imports/clipping/Clipping.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import actions from './actions';
+
+class Clipping extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      percentile: [0.9, 0.925, 0.95, 0.96, 0.97, 0.98, 0.99, 0.995, 0.999, 1.0],
```

Comment:
this should be a global object. 
```
const percentileArray = [0.9, 0.925] 
  class Clipping extends Component {
```

Reply:
ref: https://medium.freecodecamp.org/where-do-i-belong-a-guide-to-saving-react-component-data-in-state-store-static-and-this-c49b335e2a00